### PR TITLE
test: delete accoun widget testing

### PR DESCRIPTION
**Pull request by MarcosTort**

### CODE REVIEW

File: somnio_starter/__brick__/{{project_name.snakeCase()}}/lib/authentication/delete_account/view/delete_account_view.dart [hunk 1]
```diff
@@ -19,18 +19,18 @@ class DeleteAccountView extends StatelessWidget {
...
+          child: DeleteAccountForm(),
         ),
       ),
     );
   }
 }
 
-class _DeleteAccountForm extends StatelessWidget {
-  const _DeleteAccountForm();
+class DeleteAccountForm extends StatelessWidget {
```
Comment (gianfrancopapa) [inline somnio_starter/__brick__/{{project_name.snakeCase()}}/lib/authentication/delete_account/view/delete_account_view.dart:32]: Let's have this as private

File: somnio_starter/__brick__/{{project_name.snakeCase()}}/lib/authentication/delete_account/view/delete_account_view.dart [hunk 2]
```diff
@@ -61,7 +61,7 @@ class _PasswordFieldState extends State<_PasswordField> {
       (DeleteAccountBloc bloc) => bloc.state.password,
     );
     final initialValue = password.value;
-    return {{short_name.upperCase()}}TextField.passwordTextField(
+    return UITextField.passwordTextField(
```
Comment (gianfrancopapa) [inline somnio_starter/__brick__/{{project_name.snakeCase()}}/lib/authentication/delete_account/view/delete_account_view.dart]: This should be dynamic based on short_name

File: somnio_starter/__brick__/{{project_name.snakeCase()}}/lib/authentication/delete_account/view/delete_account_view.dart [hunk 3]
```diff
@@ -79,7 +79,7 @@ class _EmailField extends StatelessWidget {
   Widget build(BuildContext context) {
     final email = context.read<DeleteAccountBloc>().state.email;
     final initialValue = email.value;
-    return {{short_name.upperCase()}}TextField.emailTextField(
+    return UITextField.emailTextField(
```
Comment (gianfrancopapa) [inline somnio_starter/__brick__/{{project_name.snakeCase()}}/lib/authentication/delete_account/view/delete_account_view.dart]: This should be dynamic based on short_name

File: somnio_starter/__brick__/{{project_name.snakeCase()}}/lib/authentication/delete_account/view/delete_account_view.dart [hunk 4]
```diff
@@ -97,7 +97,7 @@ class _DeleteAccountButton extends StatelessWidget {
   Widget build(BuildContext context) {
     final isValid =
         context.select((DeleteAccountBloc bloc) => bloc.state.valid);
-    return {{short_name.upperCase()}}OutlinedButton.primary(
+    return UIOutlinedButton.primary(
```
Comment (gianfrancopapa) [inline somnio_starter/__brick__/{{project_name.snakeCase()}}/lib/authentication/delete_account/view/delete_account_view.dart]: This should be dynamic based on short_name

File: somnio_starter/__brick__/{{project_name.snakeCase()}}/lib/authentication/delete_account/bloc/delete_account_state.dart [hunk 5]
```diff
@@ -19,10 +19,16 @@ enum DeleteAccountStatus {
...
   const DeleteAccountState({
-    this.status = DeleteAccountStatus.initial,
-    this.email = const Email.pure(),
-    this.password = const Password.pure(),
+    required this.status,
+    required this.email,
+    required this.password,
   });
+
+  const DeleteAccountState.initial() : this(
```
Comment (gianfrancopapa) [inline somnio_starter/__brick__/{{project_name.snakeCase()}}/lib/authentication/delete_account/bloc/delete_account_state.dart:27]: Is this necessary? Let's follow the same approach we did for other features

File: somnio_starter/__brick__/{{project_name.snakeCase()}}/lib/authentication/delete_account/view/delete_account_view.dart (hunk 1 above)
Comment (MarcosTort) [reply somnio_starter/__brick__/{{project_name.snakeCase()}}/lib/authentication/delete_account/view/delete_account_view.dart:32]: I need it for the widget testing. Do you think its better to move it in a widgets folder?

Comment (gianfrancopapa) [reply somnio_starter/__brick__/{{project_name.snakeCase()}}/lib/authentication/delete_account/view/delete_account_view.dart:32]: Use @visibileForTesting

File: somnio_starter/__brick__/{{project_name.snakeCase()}}/lib/authentication/delete_account/bloc/delete_account_state.dart (hunk 5 above)
Comment (MarcosTort) [reply somnio_starter/__brick__/{{project_name.snakeCase()}}/lib/authentication/delete_account/bloc/delete_account_state.dart:27]: I saw we are doing the same approach in the login feature

File: somnio_starter/__brick__/{{project_name.snakeCase()}}/lib/authentication/delete_account/view/delete_account_view.dart (hunk 1 above)
Comment (MarcosTort) [reply somnio_starter/__brick__/{{project_name.snakeCase()}}/lib/authentication/delete_account/view/delete_account_view.dart:32]: oh I didn't know that, thanks!

**Review by gianfrancopapa — APPROVED** [review]

**Pull request merged**